<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAttributeDlg</class>
 <widget class="QDialog" name="EditAttributeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit attribute</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="6">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="9">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="mGetAttrBtn">
       <property name="text">
        <string>Get Attribute</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="mCloseBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mSetAttrBtn">
       <property name="text">
        <string>Set Attribute</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="1" colspan="5">
    <widget class="QComboBox" name="mAttributeCombo"/>
   </item>
   <item row="9" column="0" colspan="9">
    <widget class="QTextBrowser" name="mValueText">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Slot list</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QLineEdit" name="mSessionText"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Attribute</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mSlotIDText"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Slot ID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Slot status</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLineEdit" name="mLoginText"/>
   </item>
   <item row="1" column="0" colspan="9">
    <widget class="QComboBox" name="mSlotsCombo"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Session</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="8">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QComboBox" name="mObjectCombo">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mLabelCombo"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="mObjectText">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mSlotsCombo</tabstop>
  <tabstop>mSlotIDText</tabstop>
  <tabstop>mSessionText</tabstop>
  <tabstop>mLoginText</tabstop>
  <tabstop>mAttributeCombo</tabstop>
  <tabstop>mValueText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
